src/Pages/settings/Preferences.tsx

```diff
@@ -34,8 +34,8 @@ const PreferencesPage = () => {
             </div>
             <div className="card flex">
                 <div className="flex f-col f-grow">
-                    <div>Enable reactions</div>
-                    <small>Reactions will be shown on every page, if disabled no reactions will be shown</small>
+                    <div>Message reaction options</div>
```

Comment:
"options" is plural but this is a single checkbox, i think "Enable Reactions" was better